Credits for 0.I

RenechCDDA opened the pull request:
#### Summary
None

#### Purpose of change
We need credits for 0.I

#### Describe the solution
Write 'em

#### Describe alternatives you've considered


#### Testing


#### Additional context
Assembled via `git shortlog --no-merges --summary --count --since="Jan 4 2024"  --before="Jul 8 2025"`

(Jan 4 2024 being 0.H's fork date, Jul 8 2025 being 0.I's fork date)

Sorted from most commits to least, appended some explanations in the name list

NOTE: Once more, this is *commit count*. There are lots of caveats when using that to count contributions.

For checking actual merged PRs we can use github, e.g. https://github.com/CleverRaven/Cataclysm-DDA/pulls?q=is%3Apr+author%3ARenechCDDA+is%3Amerged+merged%3A2024-01-04..2025-07-08 (Replace my name with the person you're searching for). This will show all merged **PRs** between the 0.H and 0.I fork dates. But as with commits, PR counting comes with their own caveats...

RenechCDDA committed:
Credits for 0.I

RenechCDDA commented:
This is not intended to be final in any way, but we're very close to releasing 0.I so we need to get started on this.

akrieger commented on data/credits/en.credits:
```diff
@@ -1,6 +1,59 @@
+95	moxian
+90	Valiant (???)
+86	LyleSY
+85	ehughsbaird
+76	Qrox
```
78 akrieger + Andrew Krieger

RenechCDDA commented:
Combined Standing-Storm's, Kevin Granade's, and Akrieger's commits to get proper totals. (I had previously excluded Storm's second commit name but not tallied them. Yes, I had dropped counting over *300* more commits and they still had the most commits by far.)

Excluded Andrei8l from the list on their request.

kevingranade commented on data/credits/en.credits:
```diff
@@ -1,6 +1,61 @@
+409	RenechCDDA
+345	Maleclypse
+296	Karol1223
+276	Kevin Granade
```
I don't need "extra credit" as it were.
```suggestion
```

kevingranade committed:
no need to give Kevin double crediting

RenechCDDA committed:
First attempt at prettifying

Standing-Storm commented on data/credits/en.credits:
```diff
@@ -1,6 +1,44 @@
+<color_yellow>RenechCDDA</color> - Improvements to NPC factions, basecamps, and vehicles.
+
+<color_white>Special thanks to:</color>
+<color_yellow>Standing-Storm</color> - Creator and maintainer of the enormous 'Mind over Matter' mod.
```
```suggestion
<color_yellow>Standing-Storm</color> - Creator and maintainer of the enormous 'Mind over Matter' mod and numerous contributions to other mods (fair folk and espers and wizards, oh my!)
```

Maleclypse commented on data/credits/en.credits:
```diff
@@ -1,6 +1,44 @@
+<color_yellow>PatrikLundell</color> - An absurd amount of bugfixes and important safety changes to mapgen, tripoint infrastructure, and other critical systems.
+<color_yellow>Anton Simakov 'GuardianDll'</color> - CDDA's newest merger, somehow they've also found the time to learn C++ and still be a big-time contributor.
+<color_yellow>RenechCDDA</color> - Improvements to NPC factions, basecamps, and vehicles.
+<color_yellow>Maleclypse</color> - Xedra Evolved maintainer and merger
```
```suggestion
<color_yellow>Maleclypse</color> - Xedra Evolved maintainer and over 2k merges this cycle
```

RenechCDDA replied:
@Maleclypse I changed the wording a bit, check the commit and re-close this conversation if you don't have any issues with it.

Holli-Git commented on data/credits/en.credits:
```diff
@@ -1,6 +1,44 @@
+<color_yellow>Anton Simakov 'GuardianDll'</color> - CDDA's newest merger, somehow they've also found the time to learn C++ and still be a big-time contributor.
+<color_yellow>RenechCDDA</color> - Improvements to NPC factions, basecamps, and vehicles.
+<color_yellow>Maleclypse</color> - Xedra Evolved maintainer and over 2k merges this cycle
+<color_yellow>Holli</color> - For their dedicated work on guns, guns, and more guns!
```
```suggestion
<color_yellow>Holli</color> - For their dedicated work on all things gun and gun related
```

Procyonae commented on data/credits/en.credits:
```diff
@@ -1,6 +1,44 @@
+<color_yellow>ShnitzelX2</color> - Electronic storage for items and cleaning up 10 years of technical debt by moving items to be loaded with generic_factory. (Don't be jealous, but they're my favorite.)
+<color_yellow>alef</color> - Keeping the game building so we can keep developing and playing.
+<color_yellow>John Candlebury</color> - Aftershock. Did you know you can own and fly a colossal spaceship in Aftershock now? Yeah.
+<color_yellow>Daniel Brooks 'db48x'</color> - For fundamental work on our beautiful new ImGui windows, the UI we've always wanted.
```
"would watch at the game UI with respect" isn't very grammatically sound but kate should defo be on here

SariusSkelrets commented on data/credits/en.credits:
```diff
@@ -1,6 +1,44 @@
+<color_yellow>John Candlebury</color> - Aftershock. Did you know you can own and fly a colossal spaceship in Aftershock now? Yeah.
+<color_yellow>Daniel Brooks 'db48x'</color> - For fundamental work on our beautiful new ImGui windows, the UI we've always wanted.
+<color_yellow>moxian</color> - Mission-critical work on build and IWYU infrastructure. They've made every developer's life easier.
+<color_yellow>ehughsbaird</color>, <color_yellow>Karol1223</color>, <color_yellow>Andrew Krieger 'akrieger'</color>, <color_yellow>mqrause</color>, and <color_yellow>SariusSkelrets</color>.
```
```suggestion
<color_yellow>SariusSkelrets</color> - Expansion and polishing of Xedra Evolved's vampire content.
<color_yellow>ehughsbaird</color>, <color_yellow>Karol1223</color>, <color_yellow>Andrew Krieger 'akrieger'</color> and <color_yellow>mqrause</color>.
```

Holli-Git commented on data/credits/en.credits:
```diff
@@ -1,6 +1,44 @@
+<color_yellow>John Candlebury</color> - Aftershock. Did you know you can own and fly a colossal spaceship in Aftershock now? Yeah.
+<color_yellow>Daniel Brooks 'db48x'</color> - For fundamental work on our beautiful new ImGui windows, the UI we've always wanted.
+<color_yellow>moxian</color> - Mission-critical work on build and IWYU infrastructure. They've made every developer's life easier.
+<color_yellow>ehughsbaird</color>, <color_yellow>Karol1223</color>, <color_yellow>Andrew Krieger 'akrieger'</color>, <color_yellow>mqrause</color>, and <color_yellow>SariusSkelrets</color>.
```
```suggestion
<color_yellow>Karol1223</color> - For all things critter, creature, and crawling. 
<color_yellow>ehughsbaird</color>, <color_yellow>Andrew Krieger 'akrieger'</color>, <color_yellow>mqrause</color>, and <color_yellow>SariusSkelrets</color>.
```
Karol is like me with guns but with critters instead

RenechCDDA replied:
Not sure about this one, I looked at their post-0.H work and it wasn't much.

RenechCDDA commented:
I also moved Standing-Storm, myself (technically I moved myself in the 'first draft') and GuardianDll to the main developers section since I guess we are uh, developers. And it would've been weird if I put myself and only myself up there.

Alternatively we could just abandon this section entirely. I think it made more sense in ye olden days when e.g. we had mlangsdorf who was the officially-noted-and-recognized "maintainer of vehicles". Nowadays we're much more freeform.

RenechCDDA committed:
Second draft, with feedback from those on the credits
+ Add art contributors

akrieger commented:
Idk maybe made more sense when there were fewer contributors. I'm also green :p but don't really care.

GuardianDll committed:
Abolish Current Main Developers section
